djangocms_attributes_field/fields.py

```diff
@@ -10,6 +10,8 @@
 from django.utils.translation import ugettext as _
 from django.utils.translation import ugettext_lazy
 
+from cms.utils.compat import DJANGO_2_2
```

Comment:
This file doesn't exists in the repo, just copy the same logic into a utils file and use that to import and this should be fine.